test/definition/ApiDefinitionFactorySpec.scala

```diff
@@ -47,12 +52,14 @@ class ApiDefinitionFactorySpec extends UnitSpec {
               Scope(
                 key = readScope,
                 name = "View your Self Assessment information",
-                description = "Allow read access to self assessment data"
+                description = "Allow read access to self assessment data",
+                confidenceLevel
               ),
               Scope(
                 key = writeScope,
                 name = "Change your Self Assessment information",
-                description = "Allow write access to self assessment data"
+                description = "Allow write access to self assessment data",
+                confidenceLevel
               )
             ),
             api = APIDefinition(
```

Comment:
No test for the confidence level has been added

As seen here https://github.com/hmrc/business-details-api/blob/8cf546ceb0251d8224dcfae38ed863b755fcace9/test/definition/ApiDefinitionFactorySpec.scala#L85-L98